expert-forecast/expert-forecast-app/app.R

```diff
@@ -55,18 +55,37 @@ deaths_data <- deaths_data %>%
     filter(week_beginning < as.Date(load_date) - 1,
```

Comment:
similarly it is nicer to call functions using package::function in code others are going to use alot. This just means that the package version of the function gets called.